include/aspect/structured_data.h

```diff
@@ -87,14 +87,41 @@ namespace aspect
          *
          * The data in @p data_table consists of a Table for each of the @p n_components components.
          *
-         * The last two arguments are rvalue references, and the function will
-         * move the data so provided into another storage location. In other
-         * words, after the call, the variables passed as the last two
-         * arguments may be empty or otherwise altered.
+         * The `coordinate_values` and `data_table` arguments are rvalue references,
+         * and the function will move the data so provided into another storage
+         * location. In other words, after the call, the variables passed as the
+         * last two arguments may be empty or otherwise altered.
```

Comment:
They are not the last arguments any longer.

Reply:
@gassmoeller: Yes, I can test it using the global tomography Ascii data, we input in our models.

Reply:
You may want to talk to @bangerth first, I think the purpose is not only to find out if the code works, but also if it actually only stores a single copy per node (instead of a single copy per process).

Reply:
It might not be trivial to see if we actually need less memory (not sure how this shows up in the top, maybe under "shared"). But running it with 2 nodes to see if it works (including the check that the values are set on each rank!), would be good to know.

Reply:
Finally, we will need to check with Intelmpi, openmpi and mpich.

Reply:
We already established that it stores only one copy with the underlying deal.II patches. I would love for someone to just try a larger model without and with this patch just to make sure that we get the same results. @alarshi: If that is something that would be easy for you to try, that would be fantastic!

Reply:
@bangerth: Absolutely! So, is the goal to run on 2 nodes and then compare the difference?

Reply:
Yes, I think that would be totally sufficient to convince ourselves that it works. Thanks for giving it a try!

Reply:
@bangerth: I am getting segmentation fault when running my models with the branch. The error message is: 
[output_screen.txt](https://github.com/geodynamics/aspect/files/6787460/output_screen.txt)
I checked that the model is running with the current master. 
The input prm file and the ascii data I used are here: https://www.dropbox.com/sh/kxr9p2cw1j9ysjo/AAAvCZJ8Jhull2zLpUES5HZEa?dl=0